### ISSUE TITLE
Default country / Preferred country

### ISSUE THREAD

**Issue by rjtubera** [Daniel-Ioannou/flutter_country_picker#61]
Hi,

Is there an option to set default or preferred countries?

Thanks

**Comment by Daniel-Ioannou** [comment]
Hi, I was planning to app option for "favorite" country which will appear at the top of the list. 
I will try to implement this within the next two weeks. 

Thanks

**Comment by IvanBabic26** [comment]
Hi!
Is there any news/progress about your favourite country selection like you have in your [currency_picker](https://pub.dev/packages/currency_picker/versions/2.0.3) package? Using both packages by the way, love your work mate! :)
 Cheers!

**Comment by Daniel-Ioannou** [comment]
Favorite released in [v2.0.15](https://pub.dev/packages/country_picker/versions/2.0.15).

`favorite` argument can be used to show countries  at the top of the list. It takes a list of country code(iso2).

**Example:** 
``` Dart
showCountryPicker(
  context: context,
  favorite: <String>['SE', 'MC'],
  onSelect: (Country country) {
    print('Select country: ${country.displayName}');
  },
);
```
